lib/scarpe/background.rb

```diff
@@ -0,0 +1,14 @@
+class Scarpe
+  module Background
+    def background(color, options = {})
+      @background = color
+    end
+
+    def style
+      styles = (super if defined?(super)) || {}
```

Comment:
This small "hack" here ensures that the order or the modules don't matter. We were getting a method missing when the call stack was something like Stack#style -> Border#style -> Background#style -> 💥, so the `defined?(super)` keep us safe :)